unix/README.md

```diff
@@ -132,10 +132,11 @@ You also can optionally capture metrics about processes running on the server:
 - Max Memory Used (MB)
 - Total Alive Time
 
-To enable capturing process metrics set the `gather_process_stats` key in `ansible.cfg` to `true`
+To enable capturing process metrics set the command-line flag for machine-stats `--process-stats` to `true`
 ```
-gather_process_stats = true
+machine-stats --process-stats true
```

Comment:
I added some debug output and it looks like `process_stats` is always true with:
- `--process-stats true`
- `--process-stats=true`
- `--process-stats false`
- `--process-stats=false`
- no `--process-stats` flag at all

Please check the [`argparse` documentation](https://docs.python.org/3/library/argparse.html) and search for `store_true` or `store_false`.